app/resources/api/v3/public/card_resource.rb

```diff
@@ -20,7 +22,7 @@ class Api::V3::Public::CardResource < JSONAPI::Resource
         has_many :printings
 
         def latest_printing_id
-          @model.printings.max_by { |p| p.date_release } ['id']
+          @model.printing_ids[0]
```

Comment:
Is this now guaranteed to be enough?

Reply:
This caused me to look at the view definition again.  I was sorting by printing id descending, but i changed to use date_release DESC instead.  Results are the same, but it communicates the intent more clearly.

Since the view has the field sorted, taking the first will be correct.